### PR TITLE
Fixed preview in Android Studio

### PR DESCRIPTION
**Pull request by AfzalivE**
Previewing in Android Studio was broken because of periods in layout IDs which it cannot convert to underscores, hence NPEs for all views inside the search view layout.

This was apparent only after the isInEditMode() check for the getActivity() calls because there's no activity in the AS preview either, that threw an IllegalStateException.

### CODE REVIEW

File: sample/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -8,6 +8,7 @@
         <item name="colorAccent">@color/colorAccent</item>
         <item name="colorControlNormal">@color/colorPrimary</item>
         <item name="android:windowBackground">@color/background</item>
+        <item name="floatingSearchViewStyle">@style/CustomFloatingSearchViewTheme</item>
```
Comment (renaudcerrato) [inline sample/src/main/res/values/styles.xml:11]: That change has nothing to do with the PR, right? Can you send another PR explaining that change?

Comment (AfzalivE) [inline sample/src/main/res/values/styles.xml:11]: This change has to do with an error that is shown in the UI editor. It goes like:

Missing styles. Is the correct theme chosen for this layout?  Use the Theme combo box above the layout to choose a different layout, or fix the theme style references.  Failed to find style 'floatingSearchViewStyle' in current theme

So had to add that style in the current theme.

**Comment by AfzalivE** [comment]
The subsequent commits today are related to rendering in the AS UI editor preview tab
